pkg/vault/contracts/authorizer/TimelockAuthorizer.sol

```diff
@@ -641,7 +679,27 @@ contract TimelockAuthorizer is IAuthorizer, ReentrancyGuard {
     }
 
     /**
-     * @notice Schedules an execution to set the delay for revoking permission over `actionId` to `newDelay`.
+     * @notice Schedules an execution to set the delay for revoking permission over `actionId` to `newDelay`. This makes
+     * it impossible to revoke permission to execute `actionId` without an immutable public on-chain committment for the
+     * revoking at least `newDelay` seconds in advance.
+     *
+     * Critical actions that are performed by smart contracts and to which there is a long term commitment (e.g. minting
+     * of BAL as part of the Liquidity Mining Program) are typically subject to said revoke delays, making it impossible
+     * to disable the system without sufficient notice.
+     *
+     * How long it will take to make this change will depend on the current and new revoke delays: if increasing by more
+     * than 5 days, then the time difference between the revoke delays must pass. Otherwise, the minimum delay change
+     * execution delay of 5 days must pass instead.
+     *
+     * Only `executors` will be able to execute the scheduled action, unless `executors` is an empty array, in which
+     * case any account can execute it.
+     *
+     * Avoid scheduling multiple revoke delay changes for the same action at the same time, as this makes it harder to
+     * reason about the state of the system. If there is already a scheduled revoke delay change and there is a desire
+     * to change the future grant delay to some other value, cancel the first scheduled change and schedule a new one.
+     *
+     * Only root can call this function, but other accounts may be granted permission to cancel the scheduled execution
+     * (including global cancelers).
```

Comment:
```suggestion
     * @notice Schedules an execution to set the delay for revoking permission over `actionId` to `newDelay`. This makes
     * it impossible to revoke permission to execute `actionId` without an immutable public on-chain commitment for the
     * revoking at least `newDelay` seconds in advance.
     *
     * Critical actions that are performed by smart contracts and to which there is a long term commitment (e.g. minting
     * of BAL as part of the Liquidity Mining Program) are typically subject to such revoke delays, making it impossible
     * to disable the system without sufficient notice.
     *
     * How long it will take to make this change will depend on the current and new revoke delays: if increasing by more
     * than 5 days, then the time difference between the revoke delays must pass. Otherwise, the minimum delay change
     * execution delay of 5 days must pass instead.
     *
     * Only `executors` will be able to execute the scheduled action, unless `executors` is an empty array, in which
     * case any account can execute it.
     *
     * Avoid scheduling multiple revoke delay changes for the same action at the same time, as this makes it harder to
     * reason about the state of the system. If there is already a scheduled revoke delay change and there is a desire
     * to change the future grant delay to some other value, cancel the first scheduled change and schedule a new one.
     *
     * Only root can call this function, but other accounts may be granted permission to cancel the scheduled execution
     * (including global cancelers).
```